src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -69,14 +82,15 @@ class FinancialDisclosureSection extends React.Component {
           </div>
 
           <div className="input-section">
-            <ErrorableCheckbox required
-                errorMessage={isValidFinancialDisclosure(this.props.data) ? '' : 'Please acknowledge this requirement'}
+            <ErrorableRadioButtons required
+                errorMessage={isValidFinancialDisclosure(this.props.data) ? '' : 'Please select either "Yes" or "No"'}
                 label="I understand VA is not currently enrolling new applicants who decline to provide their financial information unless they have other qualifying eligibility factors."
                 name="understandsFinancialDisclosure"
-                checked={this.props.data.understandsFinancialDisclosure.value}
-                onValueChange={(update) => {this.props.onStateChange('understandsFinancialDisclosure', { value: update, dirty: false });}}/>
+                options={yesNo}
+                value={this.props.data.understandsFinancialDisclosure}
```

Comment:
This might have to map to `provideFinancialInfo`. But let's see what Josh gets back to us with.